src/tests/multihost/alltests/test_ldap_password_policy.py

```diff
@@ -172,7 +172,7 @@ def test_maxage(multihost, backupsssdconf, common_sssd_setup):
         ldap_modify_ds(multihost, ldap.MOD_REPLACE, user_dn, 'userPassword', [b'Secret123'])
         client.run_command('> /var/log/secure')
         tools.clear_sssd_cache()
-        client.run_command('sh /tmp/change_user_password_while_expired.sh')
+        client.run_command('sh /tmp/change_user_password_while_expired.sh', raiseonerr=False)
```

Comment:
Why are we ignoring the failure of password change script? Could we confirm if password change is happening. Is it failing due to ssh related.

Reply:
We have nothing to do whether the bash script is passing or not, we need to see only those logs that should be generated while running the bash script .